.github/ISSUE_TEMPLATE/membership.yml

```diff
@@ -3,6 +3,10 @@ description: Request membership in a Kubernetes Org
 labels: [ "area/github-membership" ]
 title: "REQUEST: New membership for <your-GH-handle>"
 body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for filling out this membership request! Please note, if you are already part of any Kubernetes GitHub organization like kubernetes-sigs and you are filing this request to be added to kubernetes, you do not need to open this request and can add yourself directly! The org memberships are now equivalent and sponsorship is not needed to join additional Kubernetes GitHub orgs.
```

Comment:
For the benefit of new contributors, can you please elaborate on:

`you do not need to open this request and can add yourself directly!`

How can a member of another Kubernetes Github organisation add themselves directly to become a member?

Reply:
Yes, I can see that a detailed description would simplify the process. I can update the PR once we update the explanation in the community docs (see [comment below](https://github.com/kubernetes/org/pull/3692#issuecomment-1250031081)).